<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newpersonal_personal_ui</class>
 <widget class="QWidget" name="newpersonal_personal_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Personal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/Add-Male-User.png</normaloff>:/new/Add-Male-User.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Apellido Paterno :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEdit_FatherName">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Apellido Materno :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="lineEdit_MotherName">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Primer Nombre :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEdit_FirstName">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Segundo Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="lineEdit_SecondName">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dni :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Dni">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Función :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_funcion">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Funcion :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="lineEdit_funcion">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="6">
      <widget class="Line" name="line_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dirección :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pais :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Country">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Departamento :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLineEdit" name="lineEdit_Departamento">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Provincia :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Provincia">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label_25">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Distrito :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QLineEdit" name="lineEdit_Distrito">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_24">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lugar :</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Lugar">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="6">
      <widget class="Line" name="line_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Estado Civil :</string>
       </property>
      </widget>
     </item>
     <item row="10" column="5">
      <widget class="QComboBox" name="comboBox_MaritalStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Soltero/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Casado/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Divorciado/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Viudo/a</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="11" column="4">
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sexo</string>
       </property>
      </widget>
     </item>
     <item row="11" column="5">
      <widget class="QComboBox" name="comboBox_Sex">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Femenino</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Masculino</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fecha de Naciemiento :</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Donacion de Organos :</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="QDateEdit" name="dateEdit_DateofBirth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="2" colspan="2">
      <widget class="QComboBox" name="comboBox_OrganDonor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>No información</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Si</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="comboBox_tipo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Médico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Enfermera</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recepcionista</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Otro</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="lineEdit_dni_to_insert">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_Acept">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_Save">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_FatherName</tabstop>
  <tabstop>lineEdit_MotherName</tabstop>
  <tabstop>lineEdit_FirstName</tabstop>
  <tabstop>lineEdit_SecondName</tabstop>
  <tabstop>lineEdit_Dni</tabstop>
  <tabstop>comboBox_tipo</tabstop>
  <tabstop>lineEdit_funcion</tabstop>
  <tabstop>lineEdit_Country</tabstop>
  <tabstop>lineEdit_Departamento</tabstop>
  <tabstop>lineEdit_Provincia</tabstop>
  <tabstop>lineEdit_Distrito</tabstop>
  <tabstop>lineEdit_Lugar</tabstop>
  <tabstop>dateEdit_DateofBirth</tabstop>
  <tabstop>comboBox_MaritalStatus</tabstop>
  <tabstop>comboBox_OrganDonor</tabstop>
  <tabstop>comboBox_Sex</tabstop>
  <tabstop>pushButton_Acept</tabstop>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
